gopass_wrapper.sh

```diff
@@ -10,6 +10,6 @@ fi
 export PATH="$PATH:/usr/local/bin" # required on MacOS/brew
 export GPG_TTY="$(tty)"
 
-gopass jsonapi listen
+./gopass-jsonapi listen
```

Comment:
I don't think this will work everywhere. I'm not sure where this script is executed, but normally gopass-jsonapi should be installed in the `$PATH`.

Reply:
Doesn't this script get generated? When is this file even used?